test/bids_examples_eeg_rest_fmri.jsonld

```diff
@@ -0,0 +1,237 @@
+      "internalIdentifier": "sub-35",
+      "lookupLabel": "sub-35",
+      "studiedState": [
+        {
+          "@id": "_:000003"
+        }
+      ]
+    },
+    {
+      "@id": "_:000005",
+      "@type": "https://openminds.ebrains.eu/core/SubjectState",
+      "age": {
+        "@type": "https://openminds.ebrains.eu/core/QuantitativeValue",
+        "unit": {
+          "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/year"
+        }
+      },
+      "internalIdentifier": "Studied state sub-36",
+      "lookupLabel": "Studied state sub-36"
+    },
+    {
+      "@id": "_:000006",
+      "@type": "https://openminds.ebrains.eu/core/Subject",
+      "internalIdentifier": "sub-36",
+      "lookupLabel": "sub-36",
+      "studiedState": [
+        {
+          "@id": "_:000005"
+        }
+      ]
+    },
+    {
+      "@id": "_:000007",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Deligianni",
+      "givenName": "F."
+    },
+    {
+      "@id": "_:000008",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Centeno",
+      "givenName": "M."
+    },
+    {
+      "@id": "_:000009",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Carmichael",
+      "givenName": "D.W."
+    },
+    {
+      "@id": "_:000010",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Zhang",
+      "givenName": "G.H."
+    },
+    {
+      "@id": "_:000011",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Clark",
+      "givenName": "C.A."
+    },
+    {
+      "@id": "_:000012",
+      "@type": "https://openminds.ebrains.eu/core/Person",
+      "familyName": "Clayden",
+      "givenName": "J.D."
+    },
+    {
+      "@id": "_:0000013",
+      "@type": "https://openminds.ebrains.eu/core/FileRepository"
+    },
+    {
+      "@id": "_:000014",
+      "@type": "https://openminds.ebrains.eu/core/DatasetVersion",
+      "author": [
+        {
+          "@id": "_:000007"
+        },
+        {
+          "@id": "_:000008"
+        },
+        {
+          "@id": "_:000009"
+        },
+        {
+          "@id": "_:000010"
+        },
+        {
+          "@id": "_:000011"
+        },
+        {
+          "@id": "_:000012"
+        }
+      ],
+      "experimentalApproach": [
+        {
+          "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/anatomy"
+        },
+        {
+          "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/electrophysiology"
+        },
+        {
+          "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/neuroimaging"
+        },
+        {
+          "@id": "https://openminds.ebrains.eu/instances/experimentalApproach/neuralConnectivity"
+        }
+      ],
+      "repository": {
+        "@id": "_:0000013"
+      },
+      "shortName": "EEG, fMRI and NODDI at rest'",
```

Comment:
Why does the DSV have only "shortName" and the DS only "fullName". I think it would be better to fill both fields for both the DSV and the DS.

Reply:
Completely agreed. I would move it to a issue to make a pull request for it latter.